src/signals/incident/components/form/ContainerSelect/components/ContainerEditList/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback } from 'react';
+import type { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { themeSpacing } from '@amsterdam/asc-ui';
+import { Close } from '@amsterdam/asc-assets';
+
+import Button from 'components/Button';
+import type { FeatureType, Item } from '../../types';
+
+const DEFAULT_ICON_SIZE = 40;
+const REMOVE_BUTTON_SIZE = 32;
+const REMOVE_ICON_SIZE = 12;
+const NAME = 'containerEditList';
+
+const StyledButton = styled(Button)`
+  flex-shrink: 0;
+  box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1);
+`;
+
+const List = styled.ul`
+  padding: 0;
+`;
+
+const ListItem = styled.li`
```

Comment:
Can we use the existing List or IconList component here?

Reply:
Yes - didn't know those were supplied by Amsterdam Styled Components